packages/effect/src/DateTime.ts

```diff
@@ -425,6 +425,22 @@ export const makeZonedFromString: (input: string) => Option.Option<Zoned> = Inte
  */
 export const now: Effect.Effect<Utc> = Internal.now
 
+/**
+ * Get the current time using the `Clock` service.
+ *
+ * @since 3.14.0
+ * @category constructors
+ * @example
+ * ```ts
+ * import { DateTime, Effect } from "effect"
+ *
+ * Effect.gen(function* () {
+ *   const now = yield* DateTime.nowAsDate
+ * })
+ * ```
+ */
+export const nowAsDate: Effect.Effect<Date> = Effect.map(Internal.now, (dateTime) => toDate(dateTime))
```

Comment:
I would use the clock service directly, no need for the indirection.

Reply:
Indeed. I did the change